### PR TITLE
chore: update playwright and @web dependencies

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
- update playwright to 1.40.1
  - requires new golden image cache
  - requires Overlay test workflow changes
  - requires Picker a11y delivery changes
  - normalizes Firefox delivery of Picker a11y model
  - requires Split Button a11y delivery updates
- update `@web` packages
  - requires stopSession package patch
  - requires commonJS plugin when building storybook

## Types of changes
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist
-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.

### CODE REVIEW

File: packages/overlay/test/index.ts [hunk 1]
```diff
@@ -429,6 +420,20 @@ export const runOverlayTriggerTests = (type: string): void => {
                     'inner click content available at point'
                 ).to.be.true;
 
+                // Why does this make the test pass in Chromium? 🤷🏻‍♂️
```
Comment (najikahalsema) [inline packages/overlay/test/index.ts:423]: How did you figure this out?

**Review by najikahalsema — APPROVED** [review]
LGTM! curious about the chromium tests passing but not a blocker. more curious how you discovered that.

File: packages/overlay/test/index.ts (hunk 1 above)
Comment (Westbrook) [reply packages/overlay/test/index.ts:423]: In the next test, it was previously using `preseSpace()` (though I also don't know why). When the test started to fail, I switched that to an actual keyboard interaction and it started to pass, so I adopted the approach here, too.

Comment (najikahalsema) [reply packages/overlay/test/index.ts:423]: Makes sense to me.... interesting. Thanks for explaining.

**Pull request merged**